server/server_test.go

```diff
@@ -1843,6 +1843,13 @@ func TestBundleScope(t *testing.T) {
 						code:   http.StatusBadRequest,
 						resp:   `{"code": "invalid_parameter", "message": "path a/b is owned by bundle \"test-bundle\""}`,
 					},
+					{
+						method: "PUT",
+						path:   "/policies/someid",
+						body:   `package other.path`,
+						code:   http.StatusBadRequest,
+						resp:   `{"code": "invalid_parameter", "message": "path x/y/z is owned by bundle \"test-bundle\""}`,
```

Comment:
So, there is a policy with id `someid`, it contains the package `x.y.z`, and we're attempting to replace this policy with one containing only the `other.path` package; effectively removing the `x.y.z` package. Am I understanding this correctly?
It might not be immediately apparent for the user that they're affecting packages not present in the pushed policy. Is there a sensible way of refining this error message? Like explicitly saying that the action would modify/remove the package.

Reply:
If the new policy was to replace an existing policy not owned by a bundle, so that the new package `other.path` is strictly additive, this action would be allowed? If so, then perhaps the error should state that the `someid` policy is protected. And this error would be reserved for when an update of an unprotected policy would infringe on a protected policy with e.g. a package overlap.

Reply:
Let's revisit the error messages for the next release. I'd rather get this issue out of the way quick. 🌻